packages/inscription/src/inscription/InscriptionView.tsx

```diff
@@ -12,16 +13,21 @@ const InscriptionView = ({ pid, ...props }: ComponentProps<typeof App>) => {
 
   const [width, setWidth] = useState(inscriptionWidthStorage().getWidth());
   const [resizeActive, setResizeActive] = useState(false);
+  const updateWidth = (delta: number) => {
+    setWidth(oldWidth => {
+      const newWidth = inscriptionWidthStorage().fixWidth(oldWidth - delta);
+      inscriptionWidthStorage().setWidth(newWidth);
+      return newWidth;
+    });
+  };
+  useHotkeys('F3', () => updateWidth(-20), { scopes: ['global'] });
+  useHotkeys('F4', () => updateWidth(20), { scopes: ['global'] });
```

Comment:
I think this will not work e.g. in eclipse... 
why not do it like the other resize panels are do this? if the focus is on the resize handle you can change the size with the arrow keys.

Reply:
resize handle can now be focused via tab.